### ISSUE TITLE
Different ways for table summary - 277

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#601]
Issue opened by Shunguo:
Based on ARIA spec https://www.w3.org/TR/wai-aria-1.2/#table, and WCAG https://www.w3.org/WAI/tutorials/tables/caption-summary/  multiple ways can be used to provide a table summary. The current engine code in https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v2/checker/accessibility/rules/table-rules.ts#L105 only checks summary attribute and aria-describedby. This is not enough, and could potentially cause false positive.   

Also see https://ibm.ent.box.com/notes/844382947910. specifically Shari's comments in the section 5 for more details.

- [ ] general rule description (SME & Dev)
- [ ]  testcases written (Dev)
- [ ]  rule bug (Dev)
- [ ]  message written (SME)
- [ ]  help written (SME)

Link to original issue:
https://github.ibm.com/ibma/accessibility-web-engine/issues/277
Image of original issue:
![image.png](https://images.zenhubusercontent.com/61bb8e6b2e4f8534832597ac/d629a981-a97e-484f-90c4-f5a1f6a87613)

**Comment by aliunwala** [comment]
Comment from Phil:
Some folks interprete that WCAG 2.1 does require a summary for complex tables:

W3C Tutorial: Caption & Summary 
https://www.w3.org/WAI/tutorials/tables/caption-summary/

"... A summary is usually only needed for complex tables." 

If we agree that a summary for complex tables is indeed required for WCAG compliance, the techniques for providing a summary are further listed in the [tutorial](https://www.w3.org/WAI/tutorials/tables/caption-summary/) that this issue/rule will be trying to check for. 

Also reference the previous box note discussion that spawned this new issue 
[E2E 201 <role logic>: table headers and rows calculation](https://ibm.ent.box.com/notes/844382947910)
and the previous Git issue 
https://github.ibm.com/IBMa/accessibility-web-engine/issues/201

**Comment by aliunwala** [comment]
We need a test case for this one. So this can be reproduced.

**Comment by aliunwala** [comment]
Waiting on test case to move this back into the triage pipeline

**Comment by tombrunet** [comment]
@mbgower Is there anything related to table summaries / descriptions that is required?

**Comment by aliunwala** [comment]
Triage:
We currently dont have table description rules anymore. Should this be closed?
